frontend/src/app/components/project-list/project-list-item/project-list-item.component.ts

```diff
@@ -1,25 +1,35 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
+import { Subject } from 'rxjs';
 import { ProjectInfoModel } from 'src/core/models/project/project-info-model';
 import { UserModel } from 'src/core/models/user/user-model';
+import { GetCurrentProjectService } from 'src/core/services/get-current-project.service';
 
 @Component({
   selector: 'app-project-list-item',
   templateUrl: './project-list-item.component.html',
   styleUrls: ['./project-list-item.component.sass']
 })
-export class ProjectListItemComponent implements OnInit {
+export class ProjectListItemComponent implements OnInit, OnDestroy {
```

Comment:
You can extend BaseComponent and not to write such basic things as `unsubscribe` and `onDestroy` function

Reply:
It wasn't used so I deleted it